src/rich-text-editor.component.js

```diff
@@ -52,13 +52,19 @@ export const RichTextEditor = forwardRef((props, editorRef) => {
 
   function emptyEditor() {
     // do it with selection and execCommand so it can be undone with Ctrl Z
-    const range = document.createRange()
-    range.selectNodeContents(divRef.current)
+    setSelection()
+    document.execCommand('removeFormat')
+    document.execCommand('delete')
+  }
+
+  function setSelection(collapse) {
```

Comment:
There might be a better name for this. `setSelection` sounds like it's a browser API or a generic function that can set any selection (such as [setSelectionRange](https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/setSelectionRange)).

```suggestion
  function selectEditor(collapse) {
```